### ISSUE TITLE
Question: colorValue

### ISSUE THREAD

**Issue by mychaelgo** [mikepenz/MaterialDrawer#608]
what item name i should override in my styles.xml if i want to change the color of toolbar header ?
I use parent theme : MaterialDrawerTheme.Light.TranslucentStatus

**Comment by mikepenz** [comment]
Which toolbar?

https://github.com/mikepenz/MaterialDrawer/blob/develop/app/src/main/res/values/styles.xml#L4

**Comment by mychaelgo** [comment]
@mikepenz  i have define toolbar.xml 

```
<?xml version="1.0" encoding="utf-8"?>
<android.support.v7.widget.Toolbar xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:background="@color/white"
    app:popupTheme="@style/ThemeOverlay.AppCompat.Light" />
```

So u want to change the title color to red

**Comment by AlpenDitrix** [comment]
@mychaelgo so you defined toolbar's color as `android:background="@color/white"` white. If you want to bind this color to theme -- then you need to use there one of previously defined colors (e.g. here https://github.com/mikepenz/MaterialDrawer/blob/develop/app/src/main/res/layout/activity_sample.xml is used `colorPrimary`)
you may link it as `?attr/` of theme or as just any `@color/`. 

Or even you may programmatically find that toolbar on activity creating and invoke `toolbar.setBackgroundColor(getResources().getColor(R.color.my_super_color));` on him

tl;dr: <Toolbar ... android:background="?attr/colorPrimary" ... />

**Comment by mychaelgo** [comment]
@AlpenDitrix  i want the background is white and the text color is red, in my phone, the background change to white but the title color in toolbar is white too

**Comment by AlpenDitrix** [comment]
@mychaelgo ah, sorry, misunderstood you

    <android.support.v7.widget.Toolbar 
        xmlns:app="http://schemas.android.com/apk/res-auto"
        ... 
        app:titleTextColor="@color/colorPrimary"
        ....
    />`

**Comment by mikepenz** [comment]
@AlpenDitrix thanks for answering
